cloudwatch_monitoring/api_calls.py

```diff
@@ -0,0 +1,198 @@
+"""
+module for making aws api calls using aws python sdk (boto3)
+
+"""
+import boto3
+
+
+class APICalls:
```

Comment:
Considering calling this `StepFunctionsApiCalls` or something. Then you can have a separate class for each of the types of AWS resources.